easy_rec/python/protos/dataset.proto

```diff
@@ -245,4 +247,6 @@ message DatasetConfig {
         HardNegativeSampler hard_negative_sampler = 103;
         HardNegativeSamplerV2 hard_negative_sampler_v2 = 104;
     }
+
+    optional HiveConfig hive_config = 27;
 }
```

Comment:
it is better to be placed in pipeline.proto where KafkaInput is placed.

Reply:
move to pipeline.proto